conga-aem-definitions/src/main/templates/aem-dispatcher-cloud/conf.d/available_vhosts/tenant.vhost.partials.hbs

```diff
@@ -34,9 +34,11 @@ ServerAlias {{this}}
 DocumentRoot "${DOCROOT}"
 # URI dereferencing algorithm is applied at Sling's level, do not decode parameters here
 AllowEncodedSlashes NoDecode
-# Add header breadcrumbs for help in troubleshooting
 <IfModule mod_headers.c>
-  Header add X-Vhost "publish"
+  # Add header breadcrumbs for help in troubleshooting
+{{#httpdCloudManagerConditional}}
+  Header add X-Vhost "{{httpd.serverName}}"
```

Comment:
It would be better to render the name of the generated file here, wdyt?

Reply:
@twolfart in addition to that this should be changed in the other roles (if affected) too.

Reply:
> It would be better to render the name of the generated file here, wdyt?

I wouldn't show internal information to the outside.

Reply:
> @twolfart in addition to that this should be changed in the other dispatcher roles (if affected) too. This should be also present in aem-dispatcher-ams.

Added.

Reply:
Agreed about the internal information, but I was thinking about a situation where the same server name is used  in different tenants. But this should be an edge case.